File: co.edu.uniandes.fixml.toCSharp/test/test3.xml/Instrument.cs

using System.Collections.Generic;
// Class for tag Instrument
class Instrument {


	List<Symbol> _Symbols = new List<Symbol>();
	List<IDSource> _IDSources = new List<IDSource>();
	List<SecurityID> _SecurityIDs = new List<SecurityID>();
	
	// default constructor
	
	public Instrument ( ) {   
		;
		
		Symbol Instrument_Symbol= new Symbol();
		this.AddSymbol( Instrument_Symbol );
		
		IDSource Instrument_IDSource= new IDSource();
		this.AddIDSource( Instrument_IDSource );
		
		SecurityID Instrument_SecurityID= new SecurityID();
		this.AddSecurityID( Instrument_SecurityID );
		
		
	}
	
	public Instrument (  ) {
	
	}
	
	public void AddSymbol( Symbol child ) {
		this._Symbols.Add( child );
	}
	
	public void AddIDSource( IDSource child ) {
		this._IDSources.Add( child );
	}
	
	public void AddSecurityID( SecurityID child ) {
		this._SecurityIDs.Add( child );
	}
	
	
} 

File: co.edu.uniandes.fixml/test/co/edu/uniandes/fixml/src-gen/test3/Java/Instrument.java
package object;

import java.util.List;
import java.util.ArrayList;

// Class for tag Instrument
public class Instrument {


  List<Symbol> Symbols = new ArrayList<Symbol>();
  List<IDSource> IDSources = new ArrayList<IDSource>();
  List<SecurityID> SecurityIDs = new ArrayList<SecurityID>();



  public Instrument ( ) {   
  
;
  
Symbol Instrument_Symbol_0= new Symbol();
this.addSymbol( Instrument_Symbol_0 );
  		
IDSource Instrument_IDSource_1= new IDSource();
this.addIDSource( Instrument_IDSource_1 );
  		
SecurityID Instrument_SecurityID_2= new SecurityID();
this.addSecurityID( Instrument_SecurityID_2 );
  		
  }
      
  public void addSymbol( Symbol child ) {
    this.Symbols.add( child );
  }
  public void addIDSource( IDSource child ) {
    this.IDSources.add( child );
  }
  public void addSecurityID( SecurityID child ) {
    this.SecurityIDs.add( child );
  }

} 